api/src/header/dto/header.dto.ts

```diff
@@ -0,0 +1,6 @@
+
+export class HeaderDto{
+  readonly name: string;
+  readonly bio: string;
+  readonly avatar: string;
```

Comment:
What about the links? 
![Screenshot from 2020-09-25 22-44-30](https://user-images.githubusercontent.com/1198975/94314429-c2f5b200-ff80-11ea-9cae-667681b115e9.png)